src/components/visualization/vegaLiteViz/vegaLiteViz.tsx

```diff
@@ -16,13 +16,36 @@ function VegaLiteViz(props: VegaLiteData) {
   const rusticTheme: Theme = useTheme()
   const isDarkTheme = rusticTheme.palette.mode === 'dark'
   const defaultFont = rusticTheme.typography.fontFamily
+  const tooltipBackgroundColor = rusticTheme.palette.primary.main
+  const tooltipTextColor = rusticTheme.palette.background.paper
+  const borderRadius = rusticTheme.shape.borderRadius
+  const tooltipFontSize = rusticTheme.typography.caption.fontSize
+  const tooltipFontWeight = rusticTheme.typography.caption.fontWeight
+  const tooltipOptions = {
```

Comment:
Should we also include `border`?

Reply:
Trying to copy MUI tooltip's styling here and it doesn't have border

Reply:
do we need to define all these variables here if they are only being used once ? I think it would be simpler to compose a object instead and use that where required.